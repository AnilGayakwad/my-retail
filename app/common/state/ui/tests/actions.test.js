
import {
  updateUiState,
} from '../actions';


describe('ui actions', () => {
  describe('updateUiState', () => {
    it('should have UI_STATE type', () => {
      const registries = {
        test: 'pass',
      };
      expect(updateUiState(registries, '123')).toMatchSnapshot();
    });
  });
});
