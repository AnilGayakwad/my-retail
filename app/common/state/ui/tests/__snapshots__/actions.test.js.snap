// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui actions updateUiState should have UI_STATE type 1`] = `
Object {
  "payload": Object {
    "test": "pass",
  },
  "type": "app/common/state/UI_STATE",
}
`;
